pkg/aggregator/aggregator.go

```diff
@@ -804,20 +799,18 @@ func (agg *BufferedAggregator) run() {
 			// we can use the aggregator to buffer them
 			agg.addOrchestratorManifest(&orchestratorManifest)
 		case event := <-agg.eventPlatformIn:
-			state := stateOk
-			tlmProcessed.Add(1, event.eventType)
-			aggregatorEventPlatformEvents.Add(event.eventType, 1)
-			err := agg.handleEventPlatformEvent(event)
-			if err != nil {
-				state = stateError
-				aggregatorEventPlatformEventsErrors.Add(event.eventType, 1)
-				log.Debugf("error submitting event platform event: %s", err)
-				if !aggregatorEventPlatformErrorLogged {
+			go func(event senderEventPlatformEvent) {
```

Comment:
Isn't the agent going to OOM with this? You're basically adding another "queue" of goroutines that's going to grow until the agent gets OOM killed.

Reply:
Thanks @djova for the comments. I'm testing around with this PR only. Agree without you, this might lead to OOM if the events are not consume fast enough by EP forwarder.

Will likely close this PR.